demos/top-app-bar_iframe/index.html

```diff
@@ -20,14 +20,15 @@
   <title>MWC Top App Bar Demo</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin>
+  <link rel="preload" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" as="style" type="text/css" crossorigin>
+  <link rel="preload" href="https://fonts.googleapis.com/css?family=Material+Icons&display=block" as="style" type="text/css" crossorigin>
   <script type="module" src="index.js"></script>
   <script nomodule src="../../node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
   <style>
     .unresolved main {
       opacity: 0;
     }
-  </style>
-  <link href="../shared/shared.css" rel="stylesheet">
+  </style><link href="../shared/shared.css" rel="stylesheet">
```

Comment:
nit: any reason to combine these two lines together?

Reply:
no, lets make them two lines.

Reply:
hmmm. i'll keep an i on batch find/replace. definitely wouldn't do this on purpose lol